### PR TITLE
Update packaging instructions

### PR DESCRIPTION
**Pull request by imobachgs**
* Update the packaging instructions.
* Add a spec for agama-cli.

### CODE REVIEW

File: PACKAGING.md [hunk 1]
```diff
@@ -4,36 +4,45 @@ Agama packages are available in the [YaST:Head:Agama project in
...
+  when building the `rubygem-agama` package.
+2. Run `bundle install` from the `service` directory.
+3. Add entries in the changes files.
+4. Open a pull request to get these changes into the repository.
+5. Once the pull request is merged, tag the repository with the proper version number. The processes
+   to build `cockpit-agama` and `agama-cli` use this information to infer the version. You can set
+   the tag with something like:
 
-      git tag --sign 0.5 --message "Version 0.5"
+      git tag --sign 1 --message "Version 1"
```
Comment (mvidner) [inline PACKAGING.md:32]: JFYI:

This looks a lot like [the instructions I have for ruby-dbus](https://github.com/mvidner/ruby-dbus/blob/c7677b911e207405e50e73fa3e93170bbf29fe8d/HOWTO-RELEASE), and I've found that I want to automate it as much as possible, which for me means writing lines that  I can paste into the terminal. Example:

```
Edit VERSION NEWS.md package/rubygem-ruby-dbus.spec

    osc vc package
    git commit -m "Releasing version $(cat VERSION)."
    rake check:changelog # implies :package
    gem2rpm --template opensuse --config package/gem2rpm.yml \
      package/ruby-dbus-$(cat VERSION).gem > package/rubygem-ruby-dbus.spec
    git commit -a --amend
    rake osc:build

Push the versioned commit; tag

    cat VERSION && git tag v$(cat VERSION)
    git push
    git push --tags
```

File: PACKAGING.md [hunk 2]
```diff
@@ -4,36 +4,45 @@ Agama packages are available in the [YaST:Head:Agama project in
...
 
 In order to release a new version, we need to:
 
-* Update the version number in the `VERSION` file in the corresponding subdirectory. These files are
-  read by the Ruby-based packages when building the gems.
-* Tag the repository with the proper number. The process to build `cockpit-agama` uses this
-  information to infer the version. You can set the tag with something like:
+1. Update the version number in the `service/VERSION` file with the new number. These file is read
+  when building the `rubygem-agama` package.
+2. Run `bundle install` from the `service` directory.
```
Comment (mvidner) [inline PACKAGING.md:25]:
```suggestion
2. `(cd service; bundle install)` # Updates Gemfile.lock which is part of the repository
```

File: PACKAGING.md [hunk 3]
```diff
@@ -4,36 +4,45 @@ Agama packages are available in the [YaST:Head:Agama project in
...
+3. Add entries in the changes files.
```
Comment (mvidner) [inline PACKAGING.md:26]:
```suggestion
3. Add entries in the changes files.
    `osc vc service/package`
    `osc vc rust/package`
    `osc vc web/package`
```

File: PACKAGING.md (hunk 1 above)
Comment (mvidner) [inline PACKAGING.md:32]:
```suggestion
      git tag --sign $(cat service/VERSION) --message "Version $(cat service/VERSION)"
```

I also see that we're tagging with a bare number. Even systemd which uses ordinal versioning pefixes the tags with a `v`, like `v256`. Just saying :)

**Review by mvidner — APPROVED** [review]
LGTM

File: PACKAGING.md (hunk 1 above)
Comment (imobachgs) [reply PACKAGING.md:32]: Good point. Actually, I think that the service is smart enough to handle the `v` prefix.

Comment (imobachgs) [reply PACKAGING.md]: I tried and it seems to work. So I have updated the instructions accordingly.

**Review by mvidner — APPROVED** [review]

**Pull request merged**